src/webaudio.js

```diff
@@ -242,6 +242,134 @@ WaveSurfer.WebAudio = {
         this.createSource();
     },
 
+    setGraph: function (source, destination) {
+    	this.analyser.disconnect();
+    	this.analyser.connect(source);
+    	destination.connect(this.gainNode);
+    },
+
```

Comment:
Right. Here's the method that will add user-defined audio nodes into the audio graph. :smile: